File: arthrData/Todo/TodoItem.cs
using System;

namespace arthrData.Todo
{
    
    public class TodoItem
    {        
        
        public int Id { get; set; }
        public string Title { get; set; }
        public bool IsDone { get; set; }
        public int Priority { get; set; }
        public string Description { get; set; }
        public DateTime? Deadline { get; set; }

        public int MasterSiteId { get; set; }                
        public virtual Core.MasterSite MasterSite { get; set; }

        public int StatusId { get; set; }
        public virtual Core.Status Status { get; set; }

    }

}

File: arthr.Web/Scripts/Models/TodoItem.ts
import { MasterSite } from './MasterSite';
import { Status } from './Status';

export class TodoItem {
    deadline: Date;
    description: string;
    isDone: boolean;
    masterSite: MasterSite;
    masterSiteId: number;
    priority: number;
    status: Status;
    statusId: number;
    title: string;
    todoItemId: number;
}
/*
    Debug Info:
    Unknown Types: MasterSite, Status
*/
